backend/chain.go

```diff
@@ -456,8 +535,12 @@ func init() {
 	daoPrivate = event.DAOPrivate
 	DAO = event.DAO
 	eventreg.Register(func() *zenaov1.EventInfo { return event.Info() })
+	social_feed.NewFeed("main", false, IsMember)
 }
 
+func IsMember(memberId string) bool {
+	return daoPrivate.Members.IsMember(memberId)
+}
```

Comment:
I think this should be in `events.Event` or `daokit.DAO`

Public functions in this realm should only be declared for things usable via `maketx call` IMO

Reply:
it's about an error i got like same as this one: https://github.com/gnolang/gno/issues/3922

Reply:
i'll look more in depth, maybe an underlying gno issue or just me that did something wrong
I'll keep you informed

Reply:
[78b5d3e](https://github.com/samouraiworld/zenao/pull/282/commits/78b5d3e4eb39246a8706d5d1c186c23741f379d5)